cluster/manifests/z-karpenter/vpa.yaml

```diff
@@ -19,6 +19,10 @@ spec:
     containerPolicies:
       - containerName: controller
         minAllowed:
-          memory: {{.Cluster.ConfigItems.karpenter_controller_memory}}
-          cpu: {{.Cluster.ConfigItems.karpenter_controller_cpu}}
+        {{ range $NodePool := .Cluster.NodePools}}
+        {{ if eq $NodePool.Name "default-master" }}
+          memory: {{ scaleQuantity (instanceTypeMemory (index .InstanceTypes 0)) 0.1 }}
+          cpu: {{ scaleQuantity (instanceTypeCPU (index .InstanceTypes 0)) 0.1 }}
```

Comment:
Note, these should be for `maxAllowed` as we want to limit the max, not set the min :)

Reply:
Ah yes. Thanks, I'll update.

Reply:
Update in the next commit.